notebooks/tm/hospitals-isochrones/02-contours-to-adminlevel.ipynb

```diff
@@ -0,0 +1,997 @@
+{
```

Comment:
<p>Add Save dataframe part and show head here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thinkingmachines/lacuna-fund-climate-health/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>